pyproject.toml

```diff
@@ -36,6 +36,7 @@ dynamic = ["version"]
 dependencies = [
   "packaging >= 19.1",
   "pyproject_hooks",
+  "variantlib @ https://github.com/wheelnext/variantlib/archive/main.tar.gz",
```

Comment:
Are these `@ https://github.com/wheelnext` still easy enough to work with? It seems like a `uv` workspace with a collection of all these forks of packages as `editable` is going to be needed at some point.

Reply:
"Easy" is not a word I'd use. It's quite annoying for development work (I basically `uv pip install -e ...` this package, then `uv pip install -e ...` variantlib to have it return to the development copy). However, it makes "release" installs easier when `variantlib` is not yet on the official index.